pcdsdevices/slits.py

```diff
@@ -438,7 +438,7 @@ class BeckhoffSlits(SlitsBase):
 
     def __init__(self, prefix, *, name, **kwargs):
         self._started_move = False
-        super().__init__(prefix, name=name, **kwargs)
+        super().__init__(prefix, name=name, nominal_aperture=3, **kwargs)
```

Comment:
Related nice explanation of this parameter is in the `LusiSlits`:

```
    ... we set a
    `nominal_aperture` for each unique device along the beamline. This is
    value is considered the smallest the slit aperture can become without
    blocking the beamline. Both the `xwidth` and the `ywidth`(height) need to
    exceed this `nominal_aperture` for the slits to be considered removed.
```

and the check is pretty simple in `SlitsBase`, where widths for the case of `BeckhoffSlits` is `xwidth` and `ywidth`:

```python
        self._inserted = (min(widths) < self.nominal_aperture.get())
```
```

Reply:
Might want to move this parameter to `happi` at some point- not particularly in a hurry to do so.